### PR TITLE
Changes made to enable print release manage API to beta customers

### PR DESCRIPTION
**Pull request by venkatnagula**
Changes made to enable print release manage API to beta customers

### CODE REVIEW

File: api-reference/beta/resources/printer.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ This resource supports:
 |defaults|[printerDefaults](printerdefaults.md)|The printer's default print settings.|
 |capabilities|[printerCapabilities](printercapabilities.md)|The capabilities of the printer.|
 |lastSeenDateTime|DateTimeOffset|The most recent dateTimeOffset when a printer interacted with Universal Print. Read-only.|
+|releases|[printRelease](printrelease.md) collection|The list of Job Release Options enabled for the printer. Currently Printer supports single release option only.|
```
Comment (nilakhan) [inline api-reference/beta/resources/printer.md:53]: Currently printer supports single release option only.

Comment (nilakhan) [reply api-reference/beta/resources/printer.md:53]: Do we want to specify that default is 'direct' release?

File: api-reference/beta/resources/printrelease.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+|Property|Type|Description|
+|:---|:---|:---|
+|releaseType|printReleaseType|Supported print release type|
+
+## printReleaseType values
+
+|Member|Value|Description|
+|:---|:---|:---|
+|direct|0|Print job is released to printer immediately after the print job is processed by Universal Print.|
+|qrCode|1|Print job is held within Universal Print until user scans the QR code and completes the release workflow.|
```
Comment (nilakhan) [inline api-reference/beta/resources/printrelease.md:28]: enum values need to be added to this page: 
https://learn.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-1.0#printduplexmode-values

File: api-reference/beta/resources/printer.md (hunk 1 above)
Comment (venkatnagula) [reply api-reference/beta/resources/printer.md:53]: you comment is to change from "Printer" to "printer" right ?

**Review by nilakhan — APPROVED** [review]

File: api-reference/beta/resources/printer.md [hunk 3]
```diff
@@ -50,6 +50,7 @@ This resource supports:
 |defaults|[printerDefaults](printerdefaults.md)|The printer's default print settings.|
 |capabilities|[printerCapabilities](printercapabilities.md)|The capabilities of the printer.|
 |lastSeenDateTime|DateTimeOffset|The most recent dateTimeOffset when a printer interacted with Universal Print. Read-only.|
+|releases|[printRelease](printrelease.md) collection|The list of Job Release Options enabled for the printer. Currently printer supports single release option and default release option value is direct printing.|
```
Comment (braedenp-msft) [inline api-reference/beta/resources/printer.md:53]: The fact that this property is called `releases` and the description starts with "The list of Job Release Options" makes me think this property name doesn't accurately reflect what it contains.

How can we make a note to revisit this name when moving to v1.0?

**Review by braedenp-msft — APPROVED** [review]
Looks good. I'd love to revisit the `releases` property and related parts of this change before finalizing the API in v1.0 though. Seems a bit off, but I may be missing something...

**Comment by Danielabom** [comment]
The author removed the "ready for review"  label for now. @venkatnagula will reach out.

**Comment by venkatnagula** [comment]
We no longer need to push this change. hence closing PR

**Pull request closed without merging**